sections/what_to_play.tex

```diff
@@ -28,3 +28,5 @@ \subsection*{Recommendations}
 \end{multicols}
 
 \includegraphics[width=\linewidth]{\art/enchanter.jpg}
+
+\vspace*{-1em}
```

Comment:
That moves the enchanter down?

Reply:
Yes. As you can see on the 'cs' comparison, the layout was not quite right, most likely does to tiny differences between spacings

Reply:
OK, nicely done.